bool use_real # If false stored images will be used. 
---
can_sort/Object[] objects
